chat-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -101,108 +105,65 @@ export function makeAddMessageToConversationRoute({
       logRequest({
         reqId,
         message: stripIndents`Request info:
-        User message: ${message}
-        Stream: ${stream}
-        IP: ${ip}
-        ConversationId: ${conversationIdString}`,
+          User message: ${message}
+          Stream: ${stream}
+          IP: ${ip}
+          ConversationId: ${conversationIdString}`,
       });
 
-      let conversationId: ObjectId;
-      try {
-        conversationId = new ObjectId(conversationIdString);
-      } catch (err) {
-        return sendErrorResponse({
-          reqId,
-          res,
-          httpStatus: 400,
-          errorMessage: "Invalid conversation ID",
-        });
-      }
-
       if (!isValidIp(ip)) {
-        return sendErrorResponse({
-          reqId,
-          res,
+        throw makeRequestError({
```

Comment:
totally didn't understand til reading this PR that you can throw any object. thought it needed to be using the `Error` class.